docs/src/index.md

```diff
@@ -14,42 +20,42 @@ taking a `Real` or an `AbstractVector` as an input and returns a
 Most likelihoods, like the [`GaussianLikelihood`](@ref), only require one latent Gaussian process.
 Passing a `Real` will therefore return a [`UnivariateDistribution`](https://juliastats.org/Distributions.jl/latest/univariate/),
 and passing an `AbstractVector{<:Real}` will return a [multivariate product of distributions](https://juliastats.org/Distributions.jl/latest/multivariate/#Product-distributions).
-```@repl
+```@repl test-repl
 f = 2.0;
-GaussianLikelihood()(f) == Normal(2.0)
-fs = [2.0, 3.0, 1.5]
-GaussianLikelihood()(fs) == Product([Normal(2.0), Normal(3.0), Normal(1.5)])
```

Comment:
why did this have to be changed ?

Reply:
same for the other docstring below

Reply:
I wanted to try something where the output would be more interesting but it ended up not being the best

Reply:
My first controlled `git revert` :smiling_face_with_tear: